src/StreamCore.h

```diff
@@ -173,6 +176,11 @@ class StreamCore :
     ProtocolResult runningHandler;
     StreamBuffer fieldAddress;
 
+    // Keep track of errors to reduce logging frequencies
+    ProtocolResult previousResult = Success;
+    time_t lastErrorTime;
+    int numberOfErrors = 0;
```

Comment:
This is C++11 only. Please make old C++ compatible: No initializers in class definition.

Reply:
Please fix this

Reply:
Apologies, other work has taken priority, I will look at this later in the week